README.md

```diff
@@ -1,2 +1,50 @@
 # sys-graylog-configurer
 Idempotent configurer for graylog
+
+## Configuration through environmental variables
+* `ADMIN_PASSWORD`: graylog api admin password
+* `EXTRACTORS`: space separated list of fields to be json parsed and ingested as indexed fields. Eg: `"kubernetes message log message_log"`
```

Comment:
How do we specify the prefix for the extracted fields?

Reply:
We don't. The prefix will be `extractor_name`+`_`. Simple and easy to predict

Reply:
As discussed, lets use `.` as extractor - field separator.